tests/ballerina-test/src/test/resources/datafiles/sql/SQLConnectorMYSQLDataFile.sql

```diff
@@ -39,13 +44,16 @@ insert into DateTimeTypes (row_id, date_type, time_type, datetime_type, timestam
   (1, '2017-02-03', '11:35:45', '2017-02-03 11:53:00', '2017-02-03 11:53:00');
 /
 insert into DataTypeTable (row_id, int_type, long_type, float_type, double_type, boolean_type, string_type,
-  numeric_type, decimal_type, real_type, tinyint_type, smallint_type, clob_type, blob_type, binary_type) values
+  numeric_type, decimal_type, real_type, tinyint_type, smallint_type, clob_type, binary_type) values
   (1, 10, 9223372036854774807, 123.34, 2139095039, TRUE, 'Hello',1234.567, 1234.567, 1234.567, 1, 5555,
- 'very long text', X'77736F322062616C6C6572696E6120626C6F6220746573742E',
-  X'77736F322062616C6C6572696E612062696E61727920746573742E');
+ 'very long text', X'77736F322062616C6C6572696E612062696E61727920746573742E');
+/
+insert into BlobTable values (1, X'77736F322062616C6C6572696E6120626C6F6220746573742E');
 /
 insert into DataTypeTable (row_id) values (2);
 /
+insert into BlobTable (row_id) values (2);
```

Comment:
Shall we place all the insert statements into BlobTable in a single place? Check other files too.